cmd/package.go

```diff
@@ -416,13 +416,13 @@ func extractPackageContent(tarReader *tar.Reader, target, pkgName string) error
 			return err
 		}
 
-		if header.Name == "meta/run.yaml" {
+		if header.Name == "/meta/run.yaml" {
 			// Prepare files with boot commands for this package.
 			data, err := ioutil.ReadAll(tarReader)
 			if err != nil {
 				return err
 			}
-			if err := persistBootCmdsIntoFiles(data, target, "", pkgName); err != nil {
+			if err := persistBootCmdsIntoFiles(data, target, "", pkgName+"-"); err != nil {
```

Comment:
I would delegate this to the one using the `pkgName`, i.e. [here](https://github.com/mikelangelo-project/capstan/pull/33/files#diff-b06fd7da8ce8a7082c8f4d9174efbb7fR645). Adding `-` here makes me wonder why.